packages/phasor/src/elements/brush/brush-element.ts

```diff
@@ -33,24 +33,42 @@ export function getBrushBoundFromPoints(
 export class BrushElement extends BaseElement {
   type = 'brush' as const;
   color = '#000000';
+  point: [number, number] = [0, 0];
```

Comment:
What is this field used for?

Reply:
If a brush element is being dragged and moved, shall we update its `point` instead of `xywh`, or both of them? If it's true, this may be an anti-pattern that breaks our convention.